src/sentry/incidents/serializers/alert_rule_trigger.py

```diff
@@ -45,13 +47,19 @@ def create(self, validated_data):
             )
             self._handle_actions(alert_rule_trigger, actions)
 
-            return alert_rule_trigger
         except forms.ValidationError as e:
             # if we fail in create_alert_rule_trigger, then only one message is ever returned
             raise serializers.ValidationError(e.error_list[0].message)
         except AlertRuleTriggerLabelAlreadyUsedError:
             raise serializers.ValidationError("This label is already in use for this alert rule")
 
+        if features.has(
+            "organizations:workflow-engine-metric-alert-processing",
```

Comment:
🤔 should we create like `features.py` in the `workflow_engine` module and export this string there?

That might be nice just so we can have a dependency graph of where we are checking this feature flag - and save us (me) from typos in the string.

---

no need to do as part of this PR, mostly just curious your thoughts on doing that.